### ISSUE TITLE
Fix focus ring behaviour for menu items

### ISSUE THREAD

**Issue by katarinazaprazna** [bratislava/olo.sk#721]
- The focus ring should be transparent to have a consistent look with the rest of focusable items

See:
![Screenshot 2024-10-21 at 14 57 38](https://github.com/user-attachments/assets/651d34ff-4bb0-4b4a-89e7-de74821608aa)
![Screenshot 2024-10-21 at 14 58 15](https://github.com/user-attachments/assets/79c451a9-692c-4ba9-b57f-6d25cd556494)
